code/README (kopia).md

```diff
@@ -0,0 +1,24 @@
+# Project Todos
+
+In this week's project was to build a to-do app using react/redux.
+We were to structure a redux store, write reducers and actions and interact with our store using selectors and dispatching actions.
+
+## The problem
+
+I started by sketching out what my app would look like, but didn´t come up with a design that i liked. So instead I found a design on canva.com and tried to imitate that, but also add something own to it. In a previous project, I had trouble understanding how to add fonts, but this week I figured it out, so I'm happy about that :)
+Every week I find the projects very difficult (this week as well) so I think it has become important for me to be happy about all the small steps I take in my progress. 
+Previous I´v also found styling a bit tricky, especially inputs. This week I put more time into it, and I'm pretty happy with the result. It was also fun to understand how to make my own svgs in Illustrator and how to add them to my project. But I have not yet figured out how to get them into a checkbox properly, so this week I "cheated" a bit and removed the box from the checkbox. But design-wise i think i works.. However, I understand that my design this week is not very user-friendly.
+I think I have become better at googling and finding solutions to my problems, I have discovered that I understand more in the articles that I find and what solutions to try.
+
+If I had more time it would have been nice to do a transition on my front page, but i could only get the transition to work one way, so i removed it.
+And also use "draggable" which I downloaded to be able to move my to-dos around on the page. 
+
+I noticed when I deployed my page on netlify that in safari my checkbox disappears, this I would have fixed if I had had more time...
```

Comment:
I viewed this on my iPhone and in safari and this was the only comment I had. Besides this it is all 🙌